### PR TITLE
Converge OS Install documentation into a single location 

### PR DESCRIPTION
**Pull request by uppalk1**
* updated ubuntu_package_installation section to remove Setup mirror info
* updated install_os section to add Setup mirror info as part of OS installation documentaion

### CODE REVIEW

**Comment by uppalk1** [comment]
Adding gitHub issue
#355
https://github.com/RackHD/RackHD/issues/355

**Comment by anhou** [comment]
resolves https://github.com/RackHD/RackHD/issues/355

File: docs/rackhd/install_os.rst [hunk 1]
```diff
@@ -3,6 +3,105 @@ OS Installation Workflow Support
 
 RackHD workflow support installing Operating System automatically from remote http repository.
 
+Configuring RackHD OS Mirrors
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```
Comment (anhou) [inline docs/rackhd/install_os.rst]: I think http://rackhd.readthedocs.io/en/latest/rackhd/install_os.html#setting-up-a-windows-os-repo  should be one part of this section.

**Comment by anhou** [comment]
By the way, the top level title of this section 'OS Installation Workflow Support'  may need to change to 'OS Installation' ? because it not only include workflow

**Comment by heckj** [comment]
👍 great work! Thank you!

**Comment by anhou** [comment]
:+1:

**Pull request merged**